### ISSUE TITLE
No Idea how to use the new version of the mod, color wheel gone, replaced with paint buckets

### ISSUE THREAD

**Issue by EmiiOFFline** [ChiselsAndBits/FlatColoredBlocks#86]
The mod has changed yet the description of it remains the same. How do we use the paint mixer? How do we use anything on this mod? Please help.

**Comment by kyraiskool** [comment]
yeah ever since the original dude retired we've gotten no updated recipes, guides on how to use blocks. nothing...

the paint mixer you just stick the two colors you wanna blend and use redstone to turn it on. it's not very intuitive. i wish there was a way to change this but at least as of the 1.19.2 version there's no config file anymore!! not ingame or in the files like oml....

the way they have it now, it's kinda not worth the download. literally can't see the blocks you can create EVEN IN CREATIVE! you have to individually mix each color.. there's no more color crafter block where you can just SEE it. The best work around I have is using the chisel and bits mod's clipboard to save the colors hough i have no idea what's the limit on how many blocks you can have on that. 

bring back the option to have all the blocks generated already... please

**Comment by CaitieeChan** [comment]
100% agree. The new change is inconvenient for creative mode players like myself.
